src/CommunityCommons/javasource/communitycommons/ORM.java

```diff
@@ -267,6 +274,9 @@ public static Boolean cloneObject(IContext c, IMendixObject source,
 			if (m instanceof MendixAutoNumber) {
 				continue;
 			}
+			if ((isFileDocument(source) || isFileDocument(target)) && "__UUID__".equals(m.getName())) {
+			  continue;
+            }
```

Comment:
Indent is wrong. It seems the rest of the file is using tabs.

Reply:
And done